### ISSUE TITLE
Add man page for sid and usid commands

### ISSUE THREAD

**Issue by prajnoha** [sid-project/sid#36]

**Comment by trgill** [comment]
Looks like sid man page is done.  https://github.com/sid-project/sid/blob/7ab9609a30e19357ea42c30af44a392ffb564e86/man/sid.8.in#L1

I'll add one for usid.

**Comment by trgill** [comment]
Just thinking about this.... do we want users to be able to invoke usid?

While testing - I tried stuff like:

```
# ./usr/lib/udev/usid scan
<usid> Failed to get value for SEQNUM key from environment: Required key not available.
```

At a quick glance, it looks like usid is dependent on being invoked via  a udev context.  Is that correct?

**Comment by prajnoha** [comment]
Yes, indeed, `usid` is supposed to be executed from within udev rules only. At first, I placed it in `/usr/sbin` just like other binaries, but then I moved that to the udev specific directory instead (`/usr/lib/udev/usid`) because I don't expect users to call this directly on command line (....we can use that in testsuite though to simulate incoming uevents, but that's for our own use then anyway). 

So from this perspective, we don't officially need a man page for it (I know that `rpmdiff` fusses about missing man pages for binaries placed in /bin and /sbin). So it's up to us then whether we provide the man page here or not, just for documentation...

(One nice day when the `usid` is settled down and we have everything working, hopefully, we can think of making `usid` directly a udev builtin command.)
